release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+
+.Cluster-wide Off Example
+----
+admissionConfig:
+  pluginConfig:
+    PodTolerationRestriction:
+      configuration:
+        kind: DefaultAdmissionConfig
+        apiVersion: v1
+        disable: true
+----
+
+.Cluster-wide On Example
+----
+admissionConfig:
+  pluginConfig:
+    PodTolerationRestriction:
+      configuration:
+        apiVersion: podtolerationrestriction.admission.k8s.io/v1alpha1
+        kind: Configuration
+        default:
+         - key: key3
+           value: value3
+        whitelist:
+         - key: key1
+           value: value1
+         - key: key3
+           value: value3
+----
+
+.Namespece-specific Example
+----
+piVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: ""
+    openshift.io/display-name: ""
+    openshift.io/sa.scc.mcs: s0:c8,c7
+    openshift.io/sa.scc.supplemental-groups: 1000070000/10000
+    openshift.io/sa.scc.uid-range: 1000070000/10000
+    scheduler.alpha.kubernetes.io/defaultTolerations: '[ { "key": "key1", "value":"value1" }]'
+    scheduler.alpha.kubernetes.io/tolerationsWhitelist: '[ { "key": "key1", "value":
+      "value1" }, { "key": "key2", "value": "value2" } ]'
+  generateName: dma-
+spec:
+  finalizers:
+  - openshift.io/origin
+  - kubernetes
+----
+
+[[ocp-37-security]]
+=== Security
+
+[[ocp-37-documented-private-public-key-configurations-and-crypto-levels]]
+==== Documented Private and Public Key Configurations and Crypto Levels
+
+{product-title} is a secured by default implementation of Kubernetes.
+
+{product-title} leverages Transport Layer Security (TLS) cipher suites, JSON Web
+Algorithms (JWA) crypto algorithms, and offers external libraries such as The
+Generic Security Service Application Program Interface (GSSAPI) and libgpgme.
+
+xref:../architecture/index.adoc#architecture-index[Private and public key
+configurations and Crypto levels] are now documented for {product-title}.
+
+[[ocp-37-node-authorizer-node-restriction-admission-plug-in]]
+==== Node Authorizer and Node Restriction Admission Plug-in
+
+Pods can no longer try to gain information from secrets, configuration maps, PV,
+PVC, or API objects from other nodes.
+
+link:https://kubernetes.io/docs/admin/authorization/node/[Node authorizer]
+governs what APIs a kublet can perform. Spanning read-, write-, and auth-related
+operations. In order for the admission controller to know the identity of the
+node to enforce the rules, nodes are provisioned with credentials that identify
+them with the user name `system:node:<nodename>` and group `system:nodes`.
+
+These enforcements are in place by default on all new installations of
+{product-title} 3.7.  For upgrades from {product-title} 3.6, they are not in
+place due to the `system:nodes` RBAC being granted from OCP 3.6. To turn the
+enforcements on, run:
+
+----
+# oadm policy remove-cluster-role-from-group system:node system:nodes
+----
+
+[[ocp-37-advanced-auditing]]
```

Comment:
Missing a note (like what we had in 3.6): This feature is currently in Technology Preview and not for production workloads.